src/Sarif/Extensions/ResultExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.Extensions
+{
+    public static class ResultExtensions
```

Comment:
>ResultExtensions [](start = 24, length = 16)

One we are in the SARIF SDK, these no longer need extension methods. The generated classes in `Sarif\Autogenerated` are `partial` classes, meaning that we can extend them with first-class methods. See the classes in `Sarif\Core`. That's where these methods now belong.

Reply:
fixed!

---
In reply to: [528762018](https://github.com/microsoft/sarif-sdk/pull/2159#discussion_r528762018) [](ancestors = 528762018)